install/0001_00_cluster-version-operator_03_service.yaml

```diff
@@ -1,16 +1,19 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: cluster-version-operator
+  name: metrics
   namespace: openshift-cluster-version
   labels:
     k8s-app: cluster-version-operator
   annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: openshift-cluster-version-serving-cert
     exclude.release.openshift.io/internal-openshift-hosted: "true"
 spec:
   type: ClusterIP
   selector:
     k8s-app: cluster-version-operator
   ports:
-  - name: metrics
-    port: 9099 # chosen to be in the internal open range
+  - name: https
```

Comment:
I'd rather keep the port named `metrics`, in case the CVO somehow grows some other thing that it serves on a different port in the future.

Reply:
fixed as of 16da75689